crates/rolldown_plugin/src/plugin_driver/mod.rs

```diff
@@ -43,6 +49,12 @@ impl PluginDriver {
   ) -> SharedPluginDriver {
     let watch_files = Arc::new(DashSet::default());
     let modules = Arc::new(DashMap::default());
+    let context_load_modules = Arc::new(DashMap::default());
+    // 1024 should be enough for most cases
+    // over 1024 pending tasks are insane
+    let (tx, rx) = tokio::sync::mpsc::channel(100);
+    let tx = Arc::new(tx);
+    let rx = Arc::new(Mutex::new(rx));
```

Comment:
Maybe create a one-shot channel in place when you need it so we don't need to use `ArcMutex` to wrap it?

Reply:
here already refactor finished.